js/private/js_binary.bzl

```diff
@@ -261,6 +261,7 @@ def _bash_launcher(ctx, entry_point_path, log_prefix_rule_set, log_prefix_rule,
         "JS_BINARY__TARGET_CPU": "$(TARGET_CPU)",
         "JS_BINARY__TARGET": "$(TARGET)",
         "JS_BINARY__WORKSPACE": "$(WORKSPACE)",
+        "JS_BINARY__PACKAGE": ctx.label.package,
```

Comment:
possible that it's simpler to parse this from JS_BINARY__TARGET at the use-site?

Reply:
Personally I prefer to have in easily available for all rules in an env variable. I'll hoist the commits to change the envs into its own PR and we can discuss the merits there.